manifests/repo.pp

```diff
@@ -7,6 +7,7 @@
   Optional[String] $proxy = undef,
   Optional[String] $proxy_username = undef,
   Optional[String] $proxy_password = undef,
+  Optional[String] $aptkey_options = undef,
```

Comment:
can you please enforce the minimal string length:
```suggestion
  Optional[String[1]] $aptkey_options = undef,
```

Please also align all `=` inthe parameter section.

Reply:
The aligned `=` is a style decision that we did some time ago. Rolling changes like that out to all modules takes a lot of time. The same applies for datatypes. We want them to be as strict as possible. An empty string does not make sense here in my opinion, so we should prohibit it.